### PR TITLE
[O11y][Airflow] Add system tests for the integration package

### LINKED ISSUE #6282: [Airflow] create system test cases for the statsD data stream
Create system test cases to validate the functionality and reliability of the Airflow statsD data stream.

### PR DESCRIPTION
**Pull request by aliabbas-elastic**
## Proposed commit message

-  Added system tests for `Airflow` integration using the `observability/stream` image
- Updated the Kibana version to `8.11.0` as per [comment](https://github.com/elastic/integrations/pull/8564#issuecomment-1829137937)

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## How to test this PR locally

1. elastic-package check
2. elastic-package stack up -d -v
3. elastic-package test -v

## Related issues

- Closes #6282 

## Screenshots
image

### CODE REVIEW

**Review by harnish-elastic — APPROVED** [review]

**Comment by ritalwar** [comment]
I suggest upgrading the Kibana version to 8.11 to align with the modifications in the statsd module implementation in [Beats](https://github.com/elastic/beats/pull/36925), which are available in version 8.11. Subsequently, we should execute the tests based on the correct implementation.

**Comment by aliabbas-elastic** [comment]
> I suggest upgrading the Kibana version to 8.11 to align with the modifications in the statsd module implementation in https://github.com/elastic/beats/pull/36925, which are available in version 8.11. Subsequently, we should execute the tests based on the correct implementation.

Sure Let me update it accordingly

File: packages/airflow/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.5.2"
+  changes:
+    - description: Add system tests.
```
Comment (ritalwar) [inline packages/airflow/changelog.yml:4]:
```suggestion
    - description: Update to Kibana 8.11 to support enhanced statsd implementation, and add system test cases.
```

File: packages/airflow/changelog.yml [hunk 2]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.5.2"
```
Comment (ritalwar) [inline packages/airflow/changelog.yml:2]: Since we're upgrading Kibana version, let's opt for version 0.6.0.

**Review by ritalwar — APPROVED** [review]
LGTM!

**Pull request merged**